### DISCUSSION TITLE
Custom fonts

### DISCUSSION THREAD

**Discussion by awais019** [unocss/unocss#2877] [Q&A] [answered]
How can I use my own font files with unocss.

**Comment by zyyv** [comment, marked as answer]
Add your font-family name in uno's config. You can checnk my repo as a example.

```ts
// uno.config.ts
theme: {
  fontFamily: {
    dm: 'dm',
    craft: 'MonoCraft',
  },
}
```

Then use `font-dm`  `font-craft`.

https://github.com/zyyv/chris.me/blob/27301de55776f5359ec9c9d86b9f32d6350cdfc4/uno.config.ts#L80-L83

**Reply by awais019** [reply]
Ok but where should I define my font faces

**Reply by awais019** [reply]
Thanks I found it from your repo

**Reply by zyyv** [reply]
You can write a css file define it, or put those css into uno's config of `preflights` key. 
```ts
// uno.config.ts
preflights: [
  {
    getCSS() {
      return '
      // your font faces css
     '
    },
  },
],
```
